scripts/keybindings/util.cjs

```diff
@@ -0,0 +1,44 @@
+const keymap = {
+    backspace: "BS",
+    delete: "Del",
+    shift: "S",
+    ctrl: "C",
+};
+
+/**
+ *  Transform vscode key to vim key
+ * @param {string} key
+ */
+function key2arg(key) {
+    const parts = key
+        .toLowerCase()
+        .split("+")
+        .map((i) => (keymap[i] ? keymap[i] : i));
+    if (parts.length == 1) {
+        return `<${parts[0]}>`;
+    } else {
+        return `<${parts[0]}-${parts[1]}>`;
+    }
+}
+
+function and(...clauses) {
+    return clauses.join(" && ");
+}
+
+function or(...clauses) {
+    return clauses.join(" || ");
```

Comment:
Since && takes precedence over ||, you may need to add parentheses to ensure it works in all cases.

Reply:
Very attentive.